Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/core/check/DccTicketingServerCertificateChecker.kt

```diff
@@ -15,64 +13,7 @@ import java.security.cert.Certificate
 import javax.inject.Inject
 
 @Reusable
-class DccTicketingServerCertificateChecker @Inject constructor(
-    private val dccJWKConverter: DccJWKConverter
-) {
-
-    /**
-     * Checks the server certificate against a set of [DccJWK]
-     *
-     * @throws [DccTicketingServerCertificateCheckException] if the check fails
-     *
-     * Note that the absence of an error code indicates a successful check
-     */
-    @Throws(DccTicketingServerCertificateCheckException::class)
-    fun checkCertificate(response: Response, jwkSet: Set<DccJWK>) = checkCertificate(
-        certificateChain = response.serverCertificateChain,
-        jwkSet = jwkSet
-    )
-
-    /**
-     * Checks the server certificate against a set of [DccJWK]
-     *
-     * @throws [DccTicketingServerCertificateCheckException] if the check fails
-     *
-     * Note that the absence of an error code indicates a successful check
-     */
-    @Throws(DccTicketingServerCertificateCheckException::class)
-    fun checkCertificate(certificateChain: List<Certificate>, jwkSet: Set<DccJWK>) = try {
-        Timber.tag(TAG).d("checkCertificate(certificateChain=%s, jwkSet=%s)", certificateChain, jwkSet)
```

Comment:
The plan was to replace this logic with feeding the allowed list of hosts and fingerprints to the certificate pinner, which would cause an abort at the handshake rather than afterwards